Cargo.lock

```diff
@@ -1934,8 +1934,16 @@ version = "0.2.6"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "ee512640fe35acbfb4bb779db6f0d80704c2cacfa2e39b601ef3e3f47d1ae4c7"
 dependencies = [
- "compiler_builtins",
  "libc",
+]
+
+[[package]]
+name = "hermit-abi"
+version = "0.3.0"
```

Comment:
We will now have all three versions of hermit-abi required at the same time.

Reply:
Oh, you are right. I will write PRs to upgrade `num_cpus` etc.